### ISSUE TITLE
htop is missing kernel options for I/O and it shows no perm

### ISSUE THREAD

**Issue by BKPepe** [openwrt/packages#12418]
Maintainer: @champtar
Environment: all, OpenWrt 19.07 and OpenWrt master

Description:

If you would like to know I/O rate of read/write for processes which helps you to know which processes are writing it to flash memory, it is necessary to enable these kernel configs:
- [IO_ACCOUNTING](https://cateee.net/lkddb/web-lkddb/TASK_IO_ACCOUNTING.html)
- [TASK_DELAY_ACCT](https://cateee.net/lkddb/web-lkddb/TASK_DELAY_ACCT.html)

Both kernel options are not set.
https://github.com/openwrt/openwrt/blob/openwrt-19.07/target/linux/generic/config-4.14#L4875
https://github.com/openwrt/openwrt/blob/openwrt-19.07/target/linux/generic/config-4.14#L4873

And this is same for OpenWrt master:
https://github.com/openwrt/openwrt/blob/master/target/linux/generic/config-5.4#L5474
https://github.com/openwrt/openwrt/blob/master/target/linux/generic/config-5.4#L5472

Otherwise, htop shows you `no perm`.
Related discussion: http://forum.tinycorelinux.net/index.php?topic=23161.0

![obrazek](https://user-images.githubusercontent.com/4096468/83950565-fecb0700-a82b-11ea-8a98-108ce6ae8ecc.png)

This prevents to fully use htop. Can you please enable it in OpenWrt main repository?

**Comment by n8v8R** [comment]
also required for full *sysstat* functionality

**Comment by jow-** [comment]
> Can you please enable it in OpenWrt main repository?

Whats the effect on the kernel image size?

**Comment by n8v8R** [comment]
@jow- ~5kB

**Comment by wulfy23** [comment]
so thats what all those noperm's are! thanks BK...

**Comment by DocMAX** [comment]
i enabled both in kernel 5.15 but still says "N/A"

**Comment by peci1** [comment]
Do you run htop with root privileges?

**Comment by DocMAX** [comment]
yes of course, there is no other than root on openwrt. i now added and compiling with these extra lines:
```
CONFIG_KERNEL_TASK_DELAY_ACCT=y
CONFIG_KERNEL_TASK_IO_ACCOUNTING=y
```
